upb/def.c

```diff
@@ -1095,16 +1139,42 @@ static void fill_fieldlayout(upb_msglayout_field *field, const upb_fielddef *f)
   }
 
   if (upb_fielddef_ismap(f)) {
-    field->mode = _UPB_MODE_MAP;
+    field->mode = _UPB_MODE_MAP | (_UPB_REP_PTR << 6);
```

Comment:
please use a named constant instead of 6

Reply:
Done.